appsTransfer/build.gradle

```diff
@@ -33,6 +33,8 @@ dependencies {
     implementation "com.android.support.constraint:constraint-layout:$rootProject.ext.constraintLayoutVersion"
     implementation "com.android.support:appcompat-v7:$rootProject.ext.appCompatVersion"
     implementation "com.android.support:recyclerview-v7:$rootProject.ext.recyclerViewVersion"
+    implementation "com.android.support:gridlayout-v7:$rootProject.ext.gridLayoutVersion"
```

Comment:
must you use it ?

Reply:
yes i prefer using it to make the view elastic